### PR TITLE
Improved person hover card

### PR DESCRIPTION
**Pull request by richardolsson**
## Description
This PR modifies the `PersonHoverCard` introduced in #614 to make some improvements.

## Screenshots
image

## Changes
* Show card on mouse enter, and hide again on mouse leave (using `Popper`)
* Only show card after a slight delay of 500ms
* Reuse `TagsList` component introduced in #625 

## Notes to reviewer
This is a proof of concept. I'm not sure if we should merge it or just use it as reference for future improvements?

## Related issues
None

### CODE REVIEW

**Comment by richardolsson** [comment]
I'm curious to hear your thoughts @ziggabyte. This is just something I threw together based on the work you already did, so it's not a big deal, just something to take my mind off other things yesterday.

It will need more work before it's merged, I think but I'm curious what you think about the direction.

**Review by ziggabyte — DISMISSED** [review]
Cool!!! It adds all the things we talked about that we wanted the person hover card to do in a vague future, and that future is now! The only changes i see is to rename the old names that contain the word "popover" and tweak the space around the TagList. There can be more things but I don't see them, so I think it's good to merge when that is changed! 👯

File: src/components/PersonHoverCard/index.tsx [hunk 1]
```diff
@@ -49,37 +44,30 @@ const PersonHoverCard: React.FunctionComponent<{ personId: number }> = ({
...
               />
-              {tags && (
-                //filter for only non-hidden tags?
-                <Box display="flex" flexDirection="row" pt={1}>
-                  {tags.map((tag, index) => (
-                    <TagChip key={index} tag={tag} />
-                  ))}
-                </Box>
-              )}
+              {tags && <TagsList isGrouped={false} tags={tags} />}
```
Comment (djbusstop) [inline src/components/PersonHoverCard/index.tsx:70]: Would be nice to have a margin top here still, so it's nicely spaced.

File: src/components/PersonHoverCard/index.tsx [hunk 2]
```diff
@@ -49,37 +44,30 @@ const PersonHoverCard: React.FunctionComponent<{ personId: number }> = ({
...
     return (
-      <Box onMouseEnter={openPopover} style={{ display: 'flex' }}>
+      <Box
+        onMouseEnter={openPopover}
+        onMouseLeave={closePopover}
+        style={{ display: 'flex' }}
+      >
         {children}
-        <Popover
+        <Popper
```
Comment (djbusstop) [inline src/components/PersonHoverCard/index.tsx:53]: Another nice to have would be a cheeky little fade in animation. Not required of course.

**Review by djbusstop — DISMISSED** [review]
Have run this locally and it's super nice! Left some comments for some "nice to haves" but have approved in case you don't feel we need to implement these now.

**Comment by djbusstop** [comment]
One other tiny change I would say would really be nice is for all the icons to be aligned with the avatar. right now the icons are a bit to the right. I think it would be beautiful.

**Comment by lowlandjuju** [comment]
Went ahead and made changes according to @djbusstop comments - hope you don't mind @richardolsson 😸

**Comment by lowlandjuju** [comment]
Also... added a copy to clipboard component, I feel this could be re-used in quite a few places....

**Comment by richardolsson** [comment]
> Went ahead and made changes according to @djbusstop comments - hope you don't mind @richardolsson 😸
> Also... added a copy to clipboard component, I feel this could be re-used in quite a few places....

Nice! It looks great. I fixed the merge conflict and internationalized the `CopyToClipboard` component, and the PR is ready for review again!

**Review by djbusstop — APPROVED** [review]
Looks good. What a team effort!

**Comment by richardolsson** [comment]
Nice! Will merge. At some point this will require some more refactoring, but I think that should wait until we've seen more uses of the component.

**Pull request merged**